packages/back-end/src/queryRunners/ExperimentIncrementalRefreshQueryRunner.ts

```diff
@@ -416,7 +410,7 @@ const startExperimentIncrementalRefreshQueries = async (
             unitsMaxTimestamp: maxTimestamp,
             experimentSettingsHash:
               getExperimentSettingsHashForIncrementalRefresh(snapshotSettings),
-            unitsDimensions: experimentDimensions.map((d) => d.id),
+            unitsDimensions: eligibleDimensions.map((d) => d.id),
```

Comment:
question: `UnitDimension` is a type we have with `type: user` but here we are using `ExperimentDimension`.

Are they interchangeable to some extent or is this some debt we should clean up?